stream-chat-android-state/src/main/java/io/getstream/chat/android/offline/plugin/state/channel/internal/ChannelMutableState.kt

```diff
@@ -446,6 +469,18 @@ internal class ChannelMutableState(
         _messages.value = messages.associateBy(Message::id)
     }
 
+    private fun cacheMessages() {
+        cachedMessages.value = mapOf()
+        cachedMessages.value = cachedMessages.value + sortedVisibleMessages.value.associateBy(Message::id)
```

Comment:
Won't `cachedMessages.value` be always empty here?

Reply:
yes, we shuld just set the value to the soretdVisibility values.